src/test/java/ibm/jceplus/junit/base/BaseTestHmacSHA1.java

```diff
@@ -209,7 +209,7 @@ public void testHmacSHA1_key3() throws Exception {
         mac.update(data_3);
         byte[] digest = mac.doFinal();
 
-        assertTrue("Mac digest did not equal expected", Arrays.equals(digest, digest_3));
+        assertTrue(Arrays.equals(digest, digest_3), "Mac digest did not equal expected");
```

Comment:
Should we use an `assertArrayEquals()` here? That applies to all Mac and Hmac tests.